examples/preact/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "example-preact",
+  "private": true,
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "storybook": "start-storybook --port 6004",
+    "build-storybook": "build-storybook",
+    "preview-storybook": "http-server storybook-static --port 6004 --silent",
+    "test": "wait-on tcp:6004 && test-storybook",
```

Comment:
Sorry, I forgot to mention there's one more change needed, a `--url` parameter, like this: https://github.com/storybookjs/builder-vite/blob/3d4b66b789ae740b1606ed7013ccbb90ee8e4f3d/examples/react-ts/package.json#L11

Because by default test-storybook looks for port 6006, which is why CI is failing still.

Reply:
Thanks, good idea.  That's tripped me up in the past as well.